packages/@okta/vuepress-theme-prose/global-components/ErrorCodes.vue

```diff
@@ -217,6 +217,11 @@ export default {
 </script>
 
 <style scoped lang="scss">
+@import '../assets/css/abstracts/_colors.scss';
+
+$border_color: map-get(map-get($colors, 'form'), 'input-border');
+$link_color: map-get(map-get($colors, 'link'), 'base');
```

Comment:
Is there any better way to do this? Are we able to use the `cv` macro?

Reply:
Same comment about the other scoped scss in *.vue files

Reply:
sorry, can't use the cv macro here. That's the best I could come with without hardcoding.